examples/redis-demo/README.md

```
@@ -0,0 +1,134 @@
+
+### Always Close the Connection
+
+```javascript
+await adapter.close();
+```
+
+## How Redis Storage Works
+
+Each document is stored as a separate Redis key in the format:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)